rclcpp/test/rclcpp/test_parameter_client.cpp

```diff
@@ -293,7 +295,7 @@ TEST_F(TestParameterClient, sync_parameter_get_parameter_types) {
   auto synchronous_client = std::make_shared<rclcpp::SyncParametersClient>(node);
   std::vector<std::string> names{"foo"};
   std::vector<rclcpp::ParameterType> parameter_types =
-    synchronous_client->get_parameter_types(names);
+    synchronous_client->get_parameter_types(names, 100ms);
```

Comment:
I think this fine (adding an explicit timeout to these for testing), but maybe we should make it really large, like 1 or 10 seconds, just because I don't want to introduce flaky behavior to the tests, as asserting it happens within 100ms (while reasonable) maybe could fail sometimes and the point of these tests really aren't to meassure performance.

Reply:
make senses to me. will be addressed.